doc/psy_layer.rst

```diff
@@ -131,3 +131,66 @@ merging, inlining, OpenMP parallelisation etc.). The tree could be
 manipulated directly, however, to simplify optimisation, a set of
 transformations are supplied. These transformations are discussed in
 the next section.
+
+Schedule visualisation
+----------------------
+
+PSyclone supports visualising a schedule in two ways. Firstly the
+`view()` method outputs textual information about the contents of a
+schedule. If we were to look at the dynamo eg6 example we would see
+the following output:
+::
+   >>> schedule.view()
+   Schedule[invoke='invoke_0' dm=True]
+       Directive[OMP parallel do]
+           Loop[type='dofs',field_space='any_space_1',it_space='dofs']
+               Call copy_field_code(z,p)
+               Call inner_product_code(res,z,rs_old)
+       GlobalSum[scalar='rs_old']
+
+The above output tells us that the invoke name for the schedule we are
+looking at is `invoke_0` and that the distributed_memory option has
+been switched on. Within the schedule is an OpenMP parallel directive
+containing a loop which itself contains two builtin calls. As the
+latter of the two builtin calls requires a reduction and distributed
+memory is switched on, PSyclone has added a GlobalSum call for the
+appropriate scalar.
+
+Secondly, the `dag()` method (standing for directed acyclic graph),
+outputs the schedule and its data dependencies. By default a file in
+dot format is output with the name ``dag`` and a file in svg format is
+output with the name ``dag.svg``. The file name can be changed using
+the ``file_name`` optional argument and the output file format can be
+changed using the ``file_format`` optional argument. The file_format
+value is simply passed on to graphviz so the graphviz documentation
+should be consulted for valid formats if svg is not required.
+::
+   >>> schedule.dag(file_name="lovely", file_format="png")
+
+.. note:: The dag method can be called from any node and will will
+          output the dag for that node and all of its children.
+
+if we were to look at the dynamo eg6 example we would see the
+following image:
+
+.. image:: dag.png
+	   :width: 256
```

Comment:
If I look at the top-level psyclone.pdf that you have generated then it is fine. If I generate my own psyclone.pdf then I still have the problem. I think we can say that it is my problem rather than yours though.